build.gradle.kts

```diff
@@ -133,3 +133,9 @@ configure<org.hildan.github.changelog.plugin.GitHubChangelogExtension> {
 tasks.withType<Test> {
     useJUnitPlatform()
 }
+
+release {
+    git {
+        requireBranch.set("master")
```

Comment:
What about `main` branches as default branch?

Reply:
This repo uses master